### ISSUE TITLE
Warning: Failed prop type: The prop `link.text` is marked as required in `CustomLink`, but its value is `undefined`.

### ISSUE THREAD

**Issue by acoard** [strapi/strapi-starter-next-corporate#7]
I just pulled down the application and followed the steps in the readme.

The `back/` folder has no errors,  but in the `front/` folder I get the following output:

```
(base) xxxxx:front xxxx$ yarn dev
yarn run v1.21.1
warning package.json: No license field
$ next
ready - started server on http://localhost:3000
info  - Loaded env from /Users/xxx/dev/tmp/strapi-starter-next-corporate/front/.env.local
event - compiled successfully
event - build page: /next/dist/pages/_error
wait  - compiling...
event - compiled successfully
event - build page: /[[...slug]]
wait  - compiling...
event - compiled successfully
Warning: Failed prop type: The prop `link.text` is marked as required in `CustomLink`, but its value is `undefined`.
    in CustomLink (at testimonials-group.js:40)
    in TestimonialsGroup (at sections.js:33)
    in Section (at sections.js:65)
    in div (at sections.js:60)
    in Sections (at [[...slug]].js:28)
    in Fragment
    in DynamicPage (at _app.js:52)
    in div (at layout.js:24)
    in div (at layout.js:16)
    in div (at layout.js:14)
    in Layout (at _app.js:51)
    in Fragment
    in MyApp
    in Unknown
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in AppContainer
```

**Comment by remidej** [comment]
It looks like your Strapi server is missing some data required by the frontend.

Have you imported the Postgres backup? Or if not, have you entered all the required data in Strapi yourself?

**Comment by acoard** [comment]
Yes, I did.  The postgres backup was imported fine. My solution at the time was just to comment out that one line for the link, and everything else has worked fine. To be clear I'm not blocked on this. Besides that one field all the data exists. I'm happy to close the issue, I was simply reporting it in case you'd find it useful.

**Comment by remidej** [comment]
Thanks for the report!

I plan on refactoring the seed data mechanism once [Templates system](https://github.com/strapi/strapi/pull/7626) is ready. So I won't fix this right now because the changes would be overwritten soon after. Since this isn't blocking, I think we can to afford wait.

I'm leaving this issue open until then.

**Comment by remidej** [comment]
You can copy & paste [this line](https://github.com/strapi/strapi-starter-next-corporate/blob/master/frontend/components/sections/testimonials-group.js#L40) in your code to fix this warning
